.github/workflows/bump-logstash.yml

```diff
@@ -0,0 +1,123 @@
+---
+name: Bump Logstash Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      logstash_version:
+        description: 'Logstash version'
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install updatecli
+        uses: updatecli/updatecli-action@v2
+
+      - uses: actions/checkout@v4
+
+      - name: Create updatecli configuration
```

Comment:
In my opinion, this snippet should be moved to a static file, then we can easily test the updatecli manifest locally without the need to use the CI.

We use that pattern across different places, for instance, I recently created https://github.com/elastic/elastic-agent/pull/9686

Reply:
Good suggestion, i'll work on that!